### ISSUE TITLE
About SkinningWeights

### ISSUE THREAD

**Issue by haoz19** [alexanderlarin/pynocchio#4]
Hello, 

Thanks for sharing this good work!

Could you share some instructions on calculating the skinning weights from the code? 

Many Thanks

**Comment by alexanderlarin** [comment]
Hi @haoz19. Sorry for the late reply, but github did not send me any notification in time.

Could you please provide detail about your question? Because as far I [understand](https://github.com/alexanderlarin/pynocchio/issues/3#issuecomment-1969864459) you just require exposed function to get weights from python code. If so it's not big deal to add it for me.

But if you interested in understanding math under the code you need to deep dive to original paper which I pointed in readme file of the project.

**Comment by haoz19** [comment]
Yeah, I just want to have some functions while I did not know C++ so it's a little bit hard for me. However, thanks to your binddings.cpp file and some valuable examples, I figured out how to change them to get what I need. 

Thanks again for your reply and implementation of this work!

BTW, I also have the same issue as mentioned in issue #2 when I try to do pip install or clone first and do pip install ., while when I clone the files provided by [cansik] in issue #3 and do pip install . it worked. But I'm totally not clear what's the difference and make it work. It maybe can help people like me to install it successfully.

**Comment by alexanderlarin** [comment]
Thank you for your feedback, I'll investigate trouble with build error from #2 and notify you about the results.

If you have any suggestion about python api or need additional methods, please note me.
